azure_devops_observability/demo_blog_post/document-dashboard/azure_devops_sdlc_events.json

```diff
@@ -0,0 +1,3093 @@
+            "scale": "linear"
+          }
+        },
+        "honeycomb": {
+          "colorMode": "color-palette",
+          "colorPalette": "blue",
+          "dataMappings": {
+            "value": "count()"
+          },
+          "displayedFields": [
+            "result"
+          ],
+          "legend": {
+            "hidden": false,
+            "position": "auto"
+          },
+          "shape": "hexagon"
+        },
+        "singleValue": {
+          "alignment": "center",
+          "autoscale": true,
+          "colorThresholdTarget": "value",
+          "isIconVisible": true,
+          "label": "Builds Succeeded",
+          "prefixIcon": "SuccessIcon",
+          "recordField": "count()",
+          "showLabel": false,
+          "trend": {
+            "isVisible": false,
+            "trendType": "auto"
+          }
+        },
+        "table": {
+          "columnTypeOverrides": [
+            {
+              "fields": [
+                "content"
+              ],
+              "id": 1728585599897,
+              "value": "log-content"
+            }
+          ],
+          "columnWidths": {},
+          "enableSparklines": false,
+          "hiddenColumns": [],
+          "lineWrapIds": [],
+          "linewrapEnabled": false,
+          "monospacedFontColumns": [],
+          "monospacedFontEnabled": false,
+          "rowDensity": "condensed"
+        },
+        "thresholds": [
+          {
+            "field": "count()",
+            "id": 1,
+            "isEnabled": true,
+            "rules": [
+              {
+                "color": {
+                  "Default": "var(--dt-colors-charts-categorical-color-09-default, #649438)"
+                },
+                "comparator": "≥",
+                "id": 0,
+                "label": "",
+                "value": 0
+              },
+              {
+                "color": {
+                  "Default": "var(--dt-colors-charts-status-warning-default, #eca440)"
+                },
+                "comparator": "≥",
+                "id": 1,
+                "label": ""
+              },
+              {
+                "color": {
+                  "Default": "var(--dt-colors-charts-status-critical-default, #c4233b)"
+                },
+                "comparator": "≥",
+                "id": 2,
+                "label": ""
+              }
+            ],
+            "title": ""
+          }
+        ],
+        "unitsOverrides": [],
+        "valueBoundaries": {
+          "max": "auto",
+          "min": "auto"
+        }
+      }
+    },
+    "1": {
+      "davis": {
+        "davisVisualization": {
+          "isAvailable": true
+        },
+        "enabled": false
+      },
+      "query": "fetch events\n| filter event.kind == \"SDLC_EVENT\"\n| filter event.provider == \"azure.devops\" and matchesPhrase(event.type,\"build\") and matchesPhrase(event.status,\"finished\")\n| filter result == \"failed\"\n| summarize count(), by: {result}",
```

Comment:
result == \"failed" could be task.outcome == \"Failure\"

Please note, `task.outcome` has a couple of possible values. I don't know the input's values and whether they can be easily mapped.